app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -322,27 +343,25 @@ def parse_stack(deployment)
           :parameters     => stack_parameters(deployment),
           :resource_group => deployment.resource_group,
 
-          :orchestration_template => stack_template(deployment)
+          :orchestration_template => raw_stack_template(deployment)
         }
         return uid, new_result
       end
 
-      def stack_template(deployment)
+      def raw_stack_template(deployment)
```

Comment:
We typically reserve `raw` to indicate it's the real provider-specific thing.

This method looks more like metadata.

Maybe rename this method `stack_template_metadata`, or `stack_template_hash`.

Not sure ... naming is hard.

Whatever you settle on, rename the other `raw_template` variable names to something similar.